backend/substrapp/serializers/model.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import serializers
```

Comment:
Why do we need this?

Reply:
Rest of previous work when model file where registered directly in the view